newIDE/app/src/UI/SemiControlledMultiAutoComplete.js

```diff
@@ -53,6 +53,7 @@ export default function SemiControlledMultiAutoComplete(props: Props) {
             ) ||
             (props.optionsLimit && props.value.length >= props.optionsLimit)
           }
+          getOptionSelected={(option, value) => option.value === value.value}
```

Comment:
Interesting, why this?

Reply:
this is the warning we've seen in the console if you remember? 
a MUI autocomplete tries to know if an option is selected or not.
by default, it compares the option itself (which is not equal because it's an object)

Reply:
Great!